### ISSUE TITLE
Cannot show the result 

### ISSUE THREAD

**Issue by jinjinhong** [ermig1979/Simd#51]
I have use the example in Simd::Detection ,input an image ,The code is working properly,But the final test images were not shown.I want to know how to show it? thankyou!

**Comment by ermig1979** [comment]
The input and output images are saved in PGM image format. See here:
https://fileinfo.com/extension/pgm

**Comment by jinjinhong** [comment]
ok,but the question is the result cannot find in the file.

**Comment by ermig1979** [comment]
Oh! It really doesn't work in Debug mode. I have fixed this error. Thank you for bug report! 
I tested ROI (Region Of Interest) and the region wasn't include the center part of the image where the only one face was placed in debug mode.

**Comment by jinjinhong** [comment]
发自网易邮箱大师
Thank you for your reply. I have found the reason for this problem. If there is any further problem, I will email it to you again.
On 04/2/2018 14:02，Ihar Yermalayeu<notifications@github.com> wrote：

Oh! It really doesn't work in Debug mode. I have fixed this error. Thank you for bug report!
I tested ROI (Region Of Interest) and the region wasn't include the center part of the image where the only one face was placed in debug mode.
